autoload/airline/extensions.vim

```diff
@@ -34,7 +34,9 @@ let s:filetype_overrides = {
       \ 'vimshell': ['vimshell','%{vimshell#get_status_string()}'],
       \ }
 
-if exists(':Gina') && (v:version > 704 || (v:version == 704 && has("patch1898")))
+if airline#util#has_gina() &&
+      \ (v:version > 704 || (v:version == 704 && has("patch1898"))) &&
```

Comment:
> (v:version > 704 || (v:version == 704 && has("patch1898")))

please move this condition into `airline#util#has_gina`

Reply:
@chrisbra 

Please look this.

> https://github.com/lambdalisue/gina.vim/commit/43b3be43d02e6aedb1cd839cf1ecfb61b0fd6981

gina.vim get a new patch. The patch mean, gina.vim cannot load under `v:version < 800`.
So, `(v:version > 704 || (v:version == 704 && has("patch1898")))` don't need now.

What do you think?

Reply:
okay, than please add a check to `v:version > 800` to `airline#util#has_gina()`

Reply:
Hmm, if I add `v:version > 800`, I think this extension can't load in Vim8.0...
gina.vim can't provide a `:Gina` command if we use under vim8.0.
so, I think exists(':Gina') just enough.
What do you think?

Reply:
I don't understand. If Gina requires a Vim version > 8, than please add the `v:version >= 800` to the `util` function. If Gina works in a Vim 7.4, than move the existing `v:version > 704...` check to the `util` function.

Reply:
I fixed.
Please check.

Reply:
I mistaked. I will fix soon.

Reply:
Sorry, I fixed.
Please check this.